nvm.sh

```diff
@@ -1581,6 +1581,7 @@ nvm_install_binary() {
   local TMPDIR
   local VERSION_PATH
 
+  nvm_echo "Start to download and install ${FLAVOR} ${VERSION}"
```

Comment:
The flavor is for internal use - otherwise this will be `iojs iojs-v1.0.0` for example. To properly handle flavors, you'd need an if/else, with one message for node, and another for io.js.